// Find error


add(5);
function add(num){
    return (num + 1);
}

// func(5); -> Throws error
const func = function add(num){
    return (num + 1);
}